docs/iov-name-service/clients/01-rest-api.md

```diff
@@ -1,31 +1,35 @@
 ---
 id: rest-api
-title: REST API
-sidebar_label: REST API
+title: REST API of the IOV Name Service
+sidebar_label: REST API of the IOV Name Service
 ---
 
-You can connect to the IOV Name Service to run the [IOV Name Service Rest API](https://github.com/iov-one/bns/tree/master/cmd/bnsapi) on the top of your full node.
-
 ## Getting started
+The easiest way to query the IOV Name Service is through our REST API. On the IOV Name Service, you can resolve a starname, query a balance of an iov address and many others things.
 
-- You need first to setup a full node. You can [check this page](/docs/iov-name-service/validator/testnet) to have all the information needed.
-- Install the IOV Name Service API Rest on your full node following the [readme information](https://github.com/iov-one/bns/tree/master/cmd/bnsapi)
-
-## API
-
-You can see the full list of endpoints:
-http://exchangenet-bnsapi.iov.one/docs/index.html
+**For test purposes only**, you can query the Rest API of the IOV Name Service at this URL:
+http://bnsapi.iov.one:8000/
 
-### General
+## List of endpoints
+You can see the full list of endpoints for IOV Name Service Mainnet at this URL:
+http://bnsapi.iov.one:8000/docs/index.html#/
 
-- `GET /info`: returns information about this instance of `bnsapi`.
+## Running your own node
+Alternatively, you may prefer to run your own node and your own Rest API server. 
+1. You can read the [full node set up instructions](/docs/iov-name-service/validator/testnet). 
+2. You can read the [the REST API instructions to setup a Rest API server](https://github.com/iov-one/bns/tree/master/cmd/bnsapi).
 
-### Block
+## How to resolve a starname
+To resolve a starname, you need to query on your Rest API server the endpoint
+- `GET /username/resolve/:mystarname` returns all the information about the specific starname including list of crypto-addresses.
 
-- `GET /block/:height`: returns block details by height
+Example on the Rest API for the IOV Name Service mainnet: http://bnsapi.iov.one:8000/username/resolve/fabo*iov
```

Comment:
http://bnsapi.iov.one:8000/username/resolve/fabo*iov returns an error for me.  What version of `bnsapi` should the cluster be running.  Currently it's `iov1/bnsapi:v0.21.3`.

Reply:
I found the error.  I'll fix it.

Reply:
https://github.com/iov-one/mainnet/pull/66